How Should Your OSPO Work with InnerSource? - Proposal (#13)
rrrutledge/talks: 2024/Open Source Summit North America/proposal.md

# Session title

How Should Your OSPO Work with InnerSource?

# Description

> InnerSource is the application of open source development methodologies to company internal software development (https://innersourcecommons.org/).

Reviewer: ```suggestion
InnerSource is the application of open source development methodologies to organization's internal software development (https://innersourcecommons.org/).
```

We can focus the talk on an enterprise perspective, but I've seen other organizations (e.g., public administrations or semi-public organizations) with OSPOs or similar entities also applying InnerSource principles or having an InnerSource program. Therefore, IMHO I think it might be better to use the general term 'organizations' in the description. What do you think?

Author: Good point!

It was recently identified at the top Gartner's strategic trends for software engineering (https://gtnr.it/3tRYdVM)

> and is moving up the hype cycle (https://gtnr.it/4aX1vIa).

Reviewer: ```suggestion
and is moving up the hype cycle (https://gtnr.it/4aX1vIa). and the term Open Source Program Offices are also appearing for the first time in 2024
```

Author: Neat!

> With this attention on InnerSource, many organizations are creating formal programs to support InnerSource.

Reviewer: ```suggestion
With this attention on InnerSource, many organizations are creating formal programs to support InnerSource. Some of these programs are beginning to emerge in entities similar to OSPOs, or are facilitating collaboration among the two teams.
```

Author: Great!

> How should an OSPO relate to a company's InnerSource program?

Reviewer: ```suggestion
Why are organizations with OSPOs strengthening their relationships and investments in InnerSource? How should an OSPO relate to a company's InnerSource program?
```

Author: Thank you!

We'll review approaches taken and lessons learned based on many company's experiences as shared in the TODO group and the InnerSource Commons.
We'll summarize considerations to take into account when planning how to successfully navigate this relationship and opportunity in your own organization.

# Benefits to the Ecosystem

InnerSource is an emerging trend almost always overlaps with an OSPO when the practice arrives at an organization.
If this situation hasn't happened yet at a given organization, then it is likely coming soon.
A vanguard of organizations have worked out various approaches to managing this relationship.

> Reviewing this experience will prepare OSPO leaders in advance so that they are ready when this situation arrives in their organization.

Reviewer: ```suggestion
Reviewing this experience will prepare OSPO managers and other open source professionals working in, or alongside, OSPO-like entities (e.g., Open Source DevRels, compliance and security experts, project managers, etc.) in advance so that they are ready when this situation arrives in their organization.
```

Author: Yes!
